frontend/portals/CartItemSubscriptionInfo/index.jsx

```diff
@@ -49,17 +45,17 @@ const distillSubscriptions = (subscriptions, itemUnitPrice) => subscriptions
  * @return {JSX}
  */
 const CartItemSubscriptionInfo = ({
-  subscriptions,
+  rechargeInfo,
```

Comment:
change jsDoc to reflect the change in parameters

Reply:
Updated